### ISSUE TITLE
Optional playground support

### ISSUE THREAD

**Issue by uraimo** [JohnSundell/SwiftPlate#29]
Adding a playground to a project generated with SwiftPlate is a bit convoluted, you need to wrap it in a workspace and then create and register in the plist an _empty_ umbrella c header (that swiftplate normally doesn't include).

It would be nice to have all this done automatically when creating the project, if someone else is interested i could implement it.

**Comment by JohnSundell** [comment]
Sounds great to me! 👍 You don't really need the umbrella C header for a pure Swift project though, right?

**Comment by uraimo** [comment]
> You don't really need the umbrella C header for a pure Swift project though, right?

Correct, you need it only if you want to export something to objective-c, and for some mysterious reason to use the framework inside playgrounds.

So, i think we could have both the workspace and the header in the template and then remove them if the user doesn't care about playground support, better than generating them via code or having a separate branch with this additional stuff.

**Comment by agisilaos** [comment]
Yup I'm super interested in that feature! @uraimo!

**Comment by uraimo** [comment]
@agisilaos it's in my TODO list, will have time for this soon.
